integrationtests/scenarios/i001815-multiple-fsharp-core-dotnetcore-references/before/Project/project.json

```diff
@@ -0,0 +1,44 @@
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "emitEntryPoint": false,
+    "compilerName": "fsc",
+    "compile": {
+      "includeFiles": [
+        "Program.fs"
+      ]
+    }
+  },
+  "tools": {
+    "dotnet-compile-fsc": {
+      "version": "1.0.0-preview2-*",
+      "imports": [
+        "dnxcore50",
+        "portable-net45+win81",
+        "netstandard1.3"
+      ]
+    }
+  },
+  "frameworks": {
+    "net46": {
+      "dependencies": {
+        "FSharp.Core": "4.0.0.1",
+      },
+      "frameworkAssemblies": {
+        "mscorlib": "",
+        "System": "",
+        "System.Core": ""
+      }
+    },
```

Comment:
The two importing netstandard are not needed, so no issue. But yes, understood, thx @matthid for example